packages/graph-engine/src/nodes/string/case.ts

```diff
@@ -0,0 +1,86 @@
+import { INodeDefinition, ToInput, ToOutput } from '../../index.js';
+import { Node } from '../../programmatic/node.js';
+import { StringSchema } from '../../schemas/index.js';
+
+export enum CaseType {
```

Comment:
This can be a const enum

Reply:
maybe i dont know how but changing this to const but it makes the code more unreadable